### PR TITLE
Kovan notice

### LINKED ISSUE #1238: Get working with Kovan
In addition to adding Kovan as a menu option, we have reports that MetaMask currently fails signing kovan transactions.  This needs to be addressed.

### PR DESCRIPTION
**Pull request by 2-am-zzz**
Finally resolves and fixes #1238 by posting a proper notice. Also fixes a slight bug where the continue button for subsequent notices wouldn't re-disable itself after approving the first notice.

### CODE REVIEW

File: notices/notice-nonce.json [hunk 1]
```diff
@@ -1 +1 @@
-1
+2
```
Comment (flyswatter) [inline notices/notice-nonce.json]: Why is this a thing we have to manage?

File: ui/app/components/notice.js [hunk 2]
```diff
@@ -99,7 +99,10 @@ Notice.prototype.render = function () {
 
       h('button', {
         disabled,
-        onClick: onConfirm,
+        onClick: () => {
+          this.setState({disclaimerDisabled: true})
```
Comment (flyswatter) [inline ui/app/components/notice.js]: Won't this disable *all* notices, not just the one being confirmed?

**Review by danfinlay — COMMENTED** [review]
A couple questions.

File: notices/notice-nonce.json (hunk 1 above)
Comment (Zanibas) [inline notices/notice-nonce.json]: If we delete notices ever, we want the ids to keep incrementing, so this was my solution to make sure we didn't re-use ids.

File: ui/app/components/notice.js (hunk 2 above)
Comment (Zanibas) [inline ui/app/components/notice.js]: This is only for the button being disabled--it merely re-disables it. Nothing to do with the status of the actual notice being stored in memory.

Comment (flyswatter) [inline ui/app/components/notice.js]: Oh ok.  That's a strangely named property, but I guess that's ok.

File: notices/notice-nonce.json (hunk 1 above)
Comment (flyswatter) [inline notices/notice-nonce.json]: I sure wish we could just store signed hashes of these, but some of them need to be signed before the first account is generated.  Unless we secretly generated a vault on first init, and used account[0] to sign them, even store them server-side.

Anyways, that's a fun idea for another time.

**Review by danfinlay — APPROVED** [review]
LGTM

**Pull request merged**